dmrpp_generator/main.py

```diff
@@ -90,29 +90,25 @@ def process(self):
         for granule in granules:
             dmrpp_files = []
             for file_ in granule['files']:
-                if not search(f"{self.processing_regex}$", file_['filename']):
-                    self.LOGGER_TO_CW.debug(f"{self.dmrpp_version}: regex {self.processing_regex} does not match filename {file_['filename']}")
+                if not search(f"{self.processing_regex}$", file_['fileName']):
```

Comment:
Note that that with this change, the regex is just checked against the file's basename, instead of the full file path. If we use `file_['key']` instead of `file_['fileName']`, that would keep the path (or at least most of it, in the S3 case it would not include the `s3://` prefix).

I decided to go with `fileName` since NSIDC's regexes were written with just the base path in mind. I wanted to point this out in case some other users make use of the path in their regex.